decidim-proposals/app/commands/decidim/proposals/admin/notify_proposal_answer.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # A command to notify about the change of the published state for a proposal.
+      class NotifyProposalAnswer < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # proposal - The proposal to write the answer for.
+        # initial_state - The proposal state before the current process.
+        def initialize(proposal, initial_state)
+          @proposal = proposal
+          @initial_state = initial_state.to_s
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :noop when the answer is not published or the state didn't changed.
+        # - :ok when everything is valid.
+        #
+        # Returns nothing.
+        def call
+          return broadcast(:noop) unless proposal.published_state? && state_changed?
```

Comment:
In Decidim usually is broadcasted an `invalid` when its not `:ok`

Reply:
Yes, but in this case it's not that it's invalid, it's just that it's not needed to do anything

Reply:
yes I'm aware that it's not really an `:invalid`, so good to go then 😄